spec/change_set_persisters/change_set_persister_spec.rb

```diff
@@ -1541,21 +1541,20 @@
         resource = FactoryBot.create_for_repository(:complete_scanned_resource, files: [file])
         file_set = Wayfinder.for(resource).members.first
         file_set.read_groups = []
-        resource.preservation_policy = "cloud"
         resource = change_set_persister.metadata_adapter.persister.save(resource: resource)
         change_set_persister.metadata_adapter.persister.save(resource: file_set)
         change_set = DynamicChangeSet.new(resource)
 
         change_set_persister.save(change_set: change_set)
         expect(PreserveChildrenJob).to have_received(:perform_later).exactly(1).times
-        expect(Preserver).to have_received(:new).exactly(2).times
+        expect(Preserver).to have_received(:new).exactly(5).times
```

Comment:
One consequence of this preserve everything approach is that preservation is triggered multiple times for FileSets added to completed resources. Once for characterization and at least once for derivative generation.

Calls to Preserver service in this test:
1. Saving the scanned resource as a completed object
1. Characterizing the attached file
1. Generating the derivative for the attached file
1. Saving the scanned resource again
1. Saving the FileSet again

Reply:
That should be rare, and only result in re-saving the JSON file yeah?

Reply:
I'd say it's probably rare. Just wanted to point it out.